vcx/ci/Jenkinsfile

```diff
@@ -118,82 +117,32 @@ def android() {
-                sh "cd sdk/vcx/libvcx/build_scripts/ios/mac"
-
-            // 8) Run the script ./mac.01.libindy.setup.sh (make sure the brew install commands are successful)
-                sh "./mac.01.libindy.setup.sh"
-
-            // 9) Restart your terminal for environment variables to take effect and cd into sdk/vcx/libvcx/build_scripts/ios/mac
-            // 10) Run the script 'source ./mac.02.libindy.env.sh'
-                sh "./mac.02.libindy.env.sh"
-
-            // 9) Restart your terminal for environment variables to take effect and cd into sdk/vcx/libvcx/build_scripts/ios/mac
-            // 11) Run the script ./mac.03.libindy.build.sh
-                sh "./mac.03.libindy.build.sh"
-
-            // 12) Run the script ./mac.04.libvcx.setup.sh
-                sh "./mac.04.libvcx.setup.sh"
+                SCRIPTS_PATH="vcx/libvcx/build_scripts/ios/mac"
+                WORK_DIR = "/Users/jenkins"
 
-            // 13) Run the script 'source ./mac.05.libvcx.env.sh'
-                sh "./mac.05.libvcx.env.sh"
+                // Build ios architectures
+                sh "source vcx/ci/scripts/iosBuild.sh"
 
-            // 14) Run the script ./mac.06.libvcx.build.sh (Test failures do not prevent the .a files from being correctly built)
-            // Check for error
-                sh "./mac.06.libvcx.build.sh"
+                // Publish to Kraken
+                TIME_VERSION=sh (script: 'echo $(date +%Y%m%d_%H%M%S)', returnStdout: true ).trim()
+                sh "mkdir ${TIME_VERSION}"
+                sh "zip -r ${TIME_VERSION}/libvcxall_${TIME_VERSION}.zip vcx/wrappers/ios/vcx/lib/libvcxall.a"
 
-            // 15) Run the script ./mac.08.libssl.libcrypto.build.sh
-                sh "./mac.08.libssl.libcrypto.build.sh"
-
-            // 16) Run the script ./mac.09.libzmq.libsodium.build.sh
-                sh "./mac.09.libzmq.libsodium.build.sh"
-
-            // 17) Run the script ./mac.10.libminiz.libsqlite3.combine.sh
-                sh "./mac.10.libminiz.libsqlite3.combine.sh"
-
-            // 18) Run the script ./mac.11.copy.static.libs.to.app.sh
-                sh "./mac.11.copy.static.libs.to.app.sh"
-
-            // 19) Run the script ./mac.12.combine.static.libs.sh libvcxall
-                sh "./mac.12.combine.static.libs.sh libvcxall"
-                */
-
-            } catch (Exception ex) {
-                currentBuild.result = "FAILED"
                 if (env.BRANCH_NAME == "master") {
-                    fail()
+                    withCredentials([usernameColonPassword(credentialsId: 'jenkins-kraken-svc', variable: 'KRAKEN_CREDENTIALS')]) {
+                        sh "find ${TIME_VERSION} -type f -name 'libvcxall_${TIME_VERSION}.zip' -exec curl -u \"${KRAKEN_CREDENTIALS}\" -X POST  https://kraken.corp.evernym.com/repo/ios/upload -F 'file=@{}' \\;"
+                    }
                 }
+            } catch (Exception ex) {
```

Comment:
However, we do need to fail the build if the Rust/VCX build fails.

Reply:
Yes, but this is for the ios build. The master/ubuntu build still fails properly.